app/gcl/professional-cloud-network-engineer/section6-network-ops-monitoring/page.css

```
@@ -0,0 +1,670 @@
+.pcne-s6-page .hero hr {
+    border: none;
+    border-top: 1px solid var(--border-soft);
+    margin: 32px 0 0 0;
+}
+
+/* ---------- Prose ---------- */
+.pcne-s6-page .main h2 {
+    font-size: 26px;
+    font-weight: 700;
+    margin: 56px 0 20px 0;
+    padding-top: 8px;
```

Stylelint (17.14.0)

[error] Duplicate property "padding-top" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)